pkgs/tools/graphics/grimshot/default.nix

```diff
@@ -0,0 +1,53 @@
+{ stdenv
+, fetchFromGitHub
+, sway-unwrapped
+, wl-clipboard
+, libnotify
+, slurp
+, grim
+, jq
+}:
+
+stdenv.mkDerivation {
+  pname = "grimshot";
+  version = "2020-05-08";
+
+  # after sway-1.5 this may be switched to sway-unwrapped.src
+  src = fetchFromGitHub {
+    owner = "swaywm";
+    repo = "sway";
+    # latest change to contrib/grimshot*
+    rev = "b1d08db5f5112ab562f89564825e3e791b0682c4";
+    sha256 = "1j9wk2m79grhjxyx8fzgcza0virlkansl2fvlvjszk526i0igri6";
+  };
+
+  buildInputs = [
+    sway-unwrapped
+    wl-clipboard
+    libnotify
+    slurp
+    grim
+    jq
+  ];
```

Comment:
Not sure if this does anything (apart from the checkPhase).

Reply:
it makes sure the runtime dependencies are present at runtime?

Reply:
That was my question. Assuming I didn't miss any changes in Nixpkgs this shouldn't do anything as `buildInputs` aren't automatically added to `$PATH` and we don't have a build system that does this for us in this case.

Reply:
i've changed to `wrapProgram` as you did in sway-contrib